src/extension/background-script/actions/lnurl/index.ts

```diff
@@ -16,6 +16,13 @@ async function lnurl(message: MessageWebLnLnurl) {
   let lnurlDetails;
   try {
     lnurlDetails = await lnurlLib.getDetails(message.args.lnurlEncoded);
+    if ("status" in lnurlDetails && "reason" in lnurlDetails) {
+      if (lnurlDetails.reason) {
```

Comment:
honestly this is still very strange code to me. 

you check if the status is in lnurlDetails and if reason is in lnurlDetails and then in another if you check if reason is truthy. 
and for that the error message in the else case is imo also wrong. 
connection errors would throw afik, and the lnurl is also not invalid because we got some error - so the lnurl itself is correct.